libs/core/src/lib/utils/directives/line-clamp/line-clamp.directive.ts

```diff
@@ -1,239 +1,240 @@
 import { coerceNumberProperty } from '@angular/cdk/coercion';
 import {
-  Directive,
-  ElementRef,
-  EventEmitter,
-  Input,
-  AfterViewInit,
-  OnDestroy,
-  OnChanges,
-  Output,
-  Renderer2
+    Directive,
```

Comment:
Is there something that was changed in this file?
If there is no changes let's don't include this file in this PR

Reply:
There was a small fix, I mentioned that in the description ;)

"Added small fix in the lineclamp directive for text input update, and fixed file indents for consistency"

Reply:
Oh, I see, thanks